src/Molecules/Footer/FooterStyled.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  display: block;
```

Comment:
display block is overwritten by display flex

Reply:
Fixed